bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -6,9 +6,9 @@ metadata:
     capabilities: Seamless Upgrades
     categories: OpenShift Optional
     certified: "false"
-    createdAt: REPLACE_DATE
+    createdAt: "2023-05-22T16:06:59Z"
```

Comment:
why is this being changed?

Reply:
In other words, this should not be changed.  This date is updated by `render_templates` in [dist-git](https://pkgs.devel.redhat.com/cgit/containers/windows-machine-config-operator-bundle/tree/render_templates?h=rhaos-4.13-rhel-9&id=b6b1019ef95ba89e9537535f5dc1cbfafba2f0a4#n60).

Reply:
It seems that all of the changes being made by make bundle aren't needed, but is mentioned in the WMCO releasing doc as part of the post release tasks. Should we keep running make bundle and just ignore the changes it makes to the files, or remove it from the script entirely?